lms/djangoapps/teams/views.py

```diff
@@ -0,0 +1,38 @@
+"""
+View methods for the course team feature.
+"""
+
+from django.shortcuts import render_to_response
+from opaque_keys.edx.keys import CourseKey
+from courseware.courses import get_course_with_access, has_access
+from django.http import Http404
+from django.conf import settings
+from student.models import CourseEnrollment
+
+
+def teams_dashboard(request, course_id):
```

Comment:
This isn't so much a comment so much as a question. Do we have guidelines on when to use function views vs. class-based views?